titanium-scanner.xcodeproj/project.pbxproj

```diff
@@ -278,8 +281,9 @@
 				GCC_WARN_UNUSED_VALUE = NO;
 				GCC_WARN_UNUSED_VARIABLE = NO;
 				INSTALL_PATH = /usr/local/lib;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
You have both iOS 11 and iOS 12 mixed up in the build settings. Please verify both project- and target-build-settings.